cmd/ci-secret-bootstrap/main.go

```diff
@@ -36,27 +40,32 @@ import (
 )
 
 type options struct {
-	dryRun         bool
-	kubeConfigPath string
-	configPath     string
-	bwUser         string
-	bwPasswordPath string
-	cluster        string
-	force          bool
-	logLevel       string
-
+	dryRun               bool
+	force                bool
+	validateBWItemsUsage bool
+
+	kubeConfigPath  string
+	configPath      string
+	bwUser          string
+	bwPasswordPath  string
+	cluster         string
+	logLevel        string
 	impersonateUser string
+	bwPassword      string
+
+	maxConcurrency int
 
-	bwPassword     string
 	secretsGetters map[string]coreclientset.SecretsGetter
 	config         secretbootstrap.Config
-
-	maxConcurrency int
+	bwAllowItems   flagutil.Strings
 }
 
 func parseOptions() options {
 	var o options
 	fs := flag.NewFlagSet(os.Args[0], flag.ExitOnError)
+	o.bwAllowItems = flagutil.NewStrings()
+	fs.Var(&o.bwAllowItems, "bw-allow-items", "The items that will be ignored when the --validate-bitwarden-items-usage is specified")
```

Comment:
Nit: Rename to `bw-allow-unused`? Also, please mention in the docs that it can be specified multiple times